tests/output/How do I use digital certificates.json

```diff
@@ -0,0 +1,185 @@
+[
+    {
+        "search": [
```

Comment:
The format is different from before. 
Desired:
```json
[
    {},
    {}
]
```
Currently:
```json
[
   {
        "search": [
            {},
            {}
        ]
   }
]
```

Reply:
done